src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -287,12 +288,99 @@ class AccessControlAttribute : public chip::app::AttributeAccessInterface
 private:
     CHIP_ERROR ReadAcl(AttributeValueEncoder & aEncoder);
     CHIP_ERROR ReadExtension(AttributeValueEncoder & aEncoder);
-    CHIP_ERROR WriteAcl(AttributeValueDecoder & aDecoder);
+    CHIP_ERROR WriteAcl(AttributeValueDecoder & aDecoder, EndpointId endpointId);
```

Comment:
This is not necessary: AccessControl is only on endpoint 0, and you can see (in the constructor for AccessControlAttribute) that this entire handler is only registered for endpoint 0, so that can be assumed instead of passed in.

Reply:
acked